Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
```

Comment:
Consider renaming the method `calculate` to follow PascalCase naming convention as `Calculate` to maintain consistency with the .NET naming guidelines.